.pre-commit-config.yaml

```diff
@@ -170,3 +171,15 @@ repos:
           - Markdown==3.4.1
         types: [text]
         files: ^(CHANGELOG\.md|res/linux/org\.mixxx\.Mixxx\.metainfo.xml)$
+      - id: ts-source-copy
+        name: ts-source-copy
+        description: "Find translations literaly copied form the source language"
+        entry: python tools/ts_source_copy_check.py
+        stages:
+          - commit
+          - manual
```

Comment:
The stage is called `pre-commit` nowadays, but let's just remove it. I see no reason why using it should be forbidden to use as a pre-push hook for example.

```suggestion
```